content/docs/setup/kubernetes/upgrade/steps/index.md

```diff
@@ -10,7 +10,9 @@ control plane and the sidecar proxies, to a new release of Istio. The upgrade
 process may install new binaries and may change configuration and API schemas.
 The upgrade process may result in service downtime. To minimize downtime,
 please ensure your Istio control plane components and your applications are
-highly available with multiple replicas.
+highly available with multiple replicas. Note, we currently only support **a
```

Comment:
this should be a warning rather than a note with the use of the `**` for bolding.

```
{{ warning }}
Citadel does not support multiple instances.  Running multiple Citadel instances may introduce race conditions and lead to system outages.
{{ /warning }}
```

Reply:
Done.